pyro/ops/gaussian.py

```diff
@@ -170,10 +171,10 @@ def condition(self, value):
         P_bb = self.precision[..., n:, n:]
         b = value
 
-        info_vec = info_a - P_ab.matmul(b.unsqueeze(-1)).squeeze(-1)
+        info_vec = info_a - matmul(P_ab, b.unsqueeze(-1)).squeeze(-1)
```

Comment:
Maybe it's better to factor this out as `_mv(mat, vec)` or `matvecmul` or sth, both to speed up scalar case and to visually clean up usage. WDYT?

Reply:
Yup, I'll do that. It can be helpful for non-scalar vector too (though I don't expect much from it :).